src/main/java/com/bettercloud/vault/response/TransitResponse.java

```diff
@@ -0,0 +1,73 @@
+package com.bettercloud.vault.response;
+
+import com.bettercloud.vault.VaultException;
+import com.bettercloud.vault.api.Transit;
+import com.bettercloud.vault.json.Json;
+import com.bettercloud.vault.json.JsonObject;
+import com.bettercloud.vault.rest.RestResponse;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+/**
+ * This class is a container for the information returned by Vault in transit API
+ * operations (e.g. encrypt, decrypt).
+ */
+public class TransitResponse extends VaultResponse {
+    public static final String CIPHER_TEXT = "ciphertext";
+    public static final String PLAIN_TEXT = "plaintext";
+
+    private String leaseId;
+    private Boolean renewable;
+    private Long leaseDuration;
+    private String result;
+
+    public TransitResponse(final RestResponse restResponse, final int retries, final Transit.transitOperations operation) {
+        super(restResponse, retries);
+        parseMetadataFields();
+        parseResponseData(operation);
+    }
+
+    public String getResult() { return result; }
+
+    public String getLeaseId() {
+        return leaseId;
+    }
+
+    public Boolean getRenewable() {
+        return renewable;
+    }
+
+    public Long getLeaseDuration() {
+        return leaseDuration;
+    }
+
+    private void parseMetadataFields() {
+        try {
+            final String jsonString = new String(getRestResponse().getBody(), StandardCharsets.UTF_8);
+            final JsonObject jsonObject = Json.parse(jsonString).asObject();
+
+            this.leaseId = jsonObject.get("lease_id").asString();
+            this.renewable = jsonObject.get("renewable").asBoolean();
+            this.leaseDuration = jsonObject.get("lease_duration").asLong();
+        } catch (Exception ignored) {
```

Comment:
I do love the naming convention of naming the Exception `ignored`... but isn't swallowing all exceptions super-dangerous?  I could see swallowing some particular parse violation, but even then I'd expect a comment indicating why that's safe to do...

Reply:
It's consistent with how bettercloud is doing the other json parsing for the rest of the lib. We're hoping that by keeping it consistent it'll be more likely to be accepted by them as a PR for their project.

Reply:
ok